features/GEM003_Unique-representation-identifier.feature

```diff
@@ -0,0 +1,11 @@
+@informal-proposition
+@GEM
+Feature: GEM003 - Unique Representation Identifer
+
+  Scenario: No Shape Representation identifier shall be used twice within the product representation of an element 
+
+      Given An IfcProductDefinitionShape
+        And The element has more than 1 instance(s) of IfcShapeRepresentation
```

Comment:
Technically speaking this is not necessary. The code and logic works also for lists of length 1.